docs/new_flux_ideas_and_guide.md

```diff
@@ -0,0 +1,985 @@
+the answers to the questions. You can close with a summary of your feelings after going through the whole thing 
+but please note your thoughts as you go along. It's very likely that you'll think something is crazy or off 
+initially, but maybe think it's ok when you look back later to write your summary. We're interested in 
+seeing this progression of your learning and thought process.
+
+You're allowed to ask questions in the private channel set up for this usability test in Slack. #flux-usability-yourname
+
+Welcome to the getting started guide for Flux. Flux is a functional scripting and query language that looks 
+very similar to Javascript in some parts, but with shortcut syntax for common queries. Its goal is to make 
+querying and working with data, and time series data in particular, easy and fast. While InfluxDB is the first 
+class data source for Flux, it also provides adaptors to other databases, third party APIs, files, and formats that 
+data can be represented in.
+
+This guide doesn't present the entirety of the API or its functionality, but a subset to convey the essential 
+concepts of the language. Go here for a [complete Flux API reference]().
+
+## Data Model
+The data model of InfluxDB consists of Buckets, which hold data. Time series data is written in 
+via the HTTP API and is represented as InfluxDB Line Protocol. It separates time series into measurements, tags, 
+fields, and a time. Here are some examples of the InfluxDB line protocol:
+
+```
+cpu,host=serverA,region=west usage_system=23.2,usage_user=55.1 1566000000
+events,app=foo description="something happened",state=1i 1566000000
+```
+
+In the first line you have a measurement named `cpu`, two tag key/value pairs of `host=serverA` and `region=west`, 
+two fields of `usage_system=23.2` and `usage_user=55.1` and a timestamp at the end of `1566000000`. Measurement 
+names and tag values are always strings. Fields can be either float64, int64, bool, or string. The second line in
+the snippet has data for the `events` measurement and shows a string and integer field value.
+
+The combination of a measurement, tags, and a specific field represent an individual time series, which are an
+ordered list of time, value pairs. For example, `cpu,host=serverA,region=west usage_system` would be one time 
+series from line 1 in the above example. You can generally think of tags as the dimensions on which you can
+query and slice and dice your time series data. However, you can also do that with fields, provided that you
+pay attention to performance and schema design considerations.
+
+With the data in InfluxDB it's important to keep this context of a time series in mind. There are different types 
+of numbers that can be recorded in time series and many operations only make sense when performed on an individual 
+series and then later combined in some other way. Here are some examples of data that is fed in a time series:
+
+### Metrics
```

Comment:
It's not clear to me why we need to describe types of metrics and events here before we start with Flux.